internal/relay/relay.go

```diff
@@ -59,6 +60,7 @@ func Send(
 		destinations = append(destinations, &v)
 	}
 	_, err = sesAPI.SendRawEmail(&ses.SendRawEmailInput{
+		ConfigurationSetName: setName,
```

Comment:
Small nitpick, this looks like `gofmt` has not been run on this file.
I recommend to always run `gofmt` on save. ☺️

Reply:
Applied `gofmt -s -w` to `internal/relay/relay.go`. 😇